api/src/routes/index.js

```diff
@@ -1,9 +1,36 @@
-var express = require("express");
-var router = express.Router();
+const express = require("express");
+const { StatusCodes } = require("http-status-codes");
+
+const { getStudentById } = require("../services/student.js");
+
+const router = express.Router();
 
 /* GET home page. */
 router.get("/", function (req, res) {
   res.send("Closegap!");
 });
 
+/* GET for testing db. */
+router.get("/students/:studentId", async (req, res) => {
+  let student;
+
+  try {
+    student = await getStudentById(parseInt(req.params.studentId));
+  } catch (e) {
+    return res.status(StatusCodes.BAD_REQUEST).json({
+      error: "database error",
+    });
+  }
+
+  if (student === null) {
+    return res.status(StatusCodes.NOT_FOUND).json({
```

Comment:
Purely for consistency, can we import from `http-status-codes` as `const statusCodes = require("http-status-codes");` and then use `statusCodes. BAD_REQUEST ` and `statusCodes. NOT_FOUND` here?